pkg/artifactory/user.go

```diff
@@ -78,22 +79,22 @@ var baseUserSchema = map[string]*schema.Schema{
 }
 
 func unpackUser(s *schema.ResourceData) User {
-	d := &ResourceData{s}
+	d := &utils.ResourceData{s}
 	return User{
-		Name:                     d.getString("name", false),
-		Email:                    d.getString("email", false),
-		Password:                 d.getString("password", false),
-		Admin:                    d.getBool("admin", false),
-		ProfileUpdatable:         d.getBool("profile_updatable", false),
-		DisableUIAccess:          d.getBool("disable_ui_access", false),
-		InternalPasswordDisabled: d.getBool("internal_password_disabled", false),
-		Groups:                   d.getSet("groups"),
+		Name:                     d.GetString("name", false),
```

Comment:
I think that this patterns should be entirely removed. `GetString("name", false)` 99% of the time it's false and I think we should remove it entirely

Reply:
I did a search and found that there a few instance where `true` is used. Same for `GetBool` and `GetInt`. I'm reluctant to flip them to `false` for the time being. I think a separate PR is needed solely for this.